charts/fluent-bit/values.yaml

```diff
@@ -37,6 +37,13 @@ service:
     # prometheus.io/path: "/api/v1/metrics/prometheus"
     # prometheus.io/port: "2020"
     # prometheus.io/scrape: "true"
+  serviceMonitor:
```

Comment:
Instead of nesting `serviceMonitor` in `service` as `service.serviceMonitor` , could we move `serviceMonitor` and it's nested values up a level to the root?

Reply:
Yes, good point